CMakeLists.txt

```
@@ -72,5 +78,7 @@ packageProject(
   INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include
   INCLUDE_DESTINATION include/${PROJECT_NAME}-${PROJECT_VERSION}
   VERSION_HEADER "${VERSION_HEADER_LOCATION}"
-  DEPENDENCIES ""
+  COMPATIBILITY ExactVersion
+  COMPATIBILITY SameMajorVersion
```

Duplicate option specified here.

```suggestion
  COMPATIBILITY SameMajorVersion
```